portfolio/src/main/webapp/script.js

```diff
@@ -39,14 +39,14 @@ function toggleVisibility(id) {
   }
 }
 
-function fixSite() {
+function makeUglySite() {
   const visuals = document.body;
   const buttons = document.getElementsByClassName('button');
-  visuals.style.fontFamily = 'Georgia';
-  visuals.style.color = '#000';
-  visuals.style.backgroundColor = 'white';
+  visuals.style.fontFamily = 'Comic Sans MS';
```

Comment:
IMO instead of adding styling inline, what I like to do to completely separate my JS files from CSS is add a class using element.classList.add("my-class"). This allows you to define the styles in the CSS class and not in the JS file. This makes it much neater.